keyboards/kbdfans/kbd75/keymaps/tucznak/config.h

```diff
@@ -0,0 +1,46 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "../../config.h"
```

Comment:
QMK's build script automatically includes all the `config.h` files in the build tree, so Line 4 is redundant.

```suggestion
```